toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -56,8 +58,8 @@ func convertToCosi(ic *ImageCustomizerParameters) error {
 	return nil
 }
 
-func buildCosiFile(sourceDir string, outputFile string, partitions []outputPartitionMetadata, verityMetadata []verityDeviceMetadata, imageUuidStr string) error {
-
+func buildCosiFile(sourceDir string, outputFile string, partitions []outputPartitionMetadata, verityMetadata []verityDeviceMetadata,
```

Comment:
nit:

```golang
func buildCosiFile(sourceDir string, outputFile string, partitions []outputPartitionMetadata,
	verityMetadata []verityDeviceMetadata, partUuidToMountPath map[string]string, imageUuidStr string,
) error {
	// Pre-compute a map for quick lookup of partition metadata by UUID
	...
```

This helps to visually distinguish between the function signature and its body.